.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "Python 3",
+  "build": {
+    "dockerfile": "Dockerfile",
+    "context": "..",
+    "args": {
+      "VARIANT": "3.10-bullseye",
+      "NODE_VERSION": "none"
```

Comment:
sorry if i was not clear, but i would leave the comments here so other developers can know how this works and what values are allowed.
what do you think? sorry 🙇

Reply:
I think that's a fair point.

If so, let me also ref to https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/introduction-to-dev-containers.